packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -107,27 +136,34 @@ function BasicColorList({
           tabIndex = -1;
         }
         return (
-          <StyledSwatch
-            key={patternAsBackground}
-            onClick={() => handleColorChange(pattern)}
-            pattern={pattern}
-            isSelected={isSelected}
-            isDisabled={isDisabled}
-            tabIndex={tabIndex}
-            title={patternAsBackground}
-          />
+          <Tooltip key={patternAsBackground} title={tooltip}>
+            <StyledSwatch
+              onClick={() => handleClick(pattern, isLocal)}
+              pattern={pattern}
+              isSelected={isSelected}
+              isDisabled={isDisabled}
+              tabIndex={tabIndex}
+              title={patternAsBackground}
+              aria-label={isEditMode ? deleteLabel : applyLabel}
```

Comment:
@barklund The label here was an accessible name before (e.g. just `#fff`). Now it supports deleting as well though. Any thoughts on what would be a better label now? `Apply local color: #fff` etc.?